src/wp-includes/script-loader.php

```diff
@@ -2702,6 +2683,63 @@ static function ( $html, $block ) use ( $block_name, $style_properties ) {
 	}
 }
 
+/**
+ * Function responsible for enqueuing the CSS files required for block styles functionality on the editor and frontend.
+ *
+ * @since 6.6.0
+ *
+ * @global WP_Styles $wp_styles
+ */
+function enqueue_block_styles_handle_assets() {
+	global $wp_styles;
+
+	$block_styles = WP_Block_Styles_Registry::get_instance()->get_all_registered();
+	foreach ( $block_styles as $block_name => $styles ) {
+		foreach ( $styles as $style_properties ) {
+			if ( ! isset( $style_properties['style_handle'] ) || ! isset( $wp_styles->registered[ $style_properties['style_handle'] ] ) ) {
+				continue;
+			}
+
+			/**
```

Comment:
```suggestion
			/*
```
Per [Multi-line comment](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#5-inline-comments) document